### ISSUE TITLE
trace_block error on block #21030627

### ISSUE THREAD

**Issue by bigjoker007** [erigontech/erigon#13091]
#### System information

Erigon version:  2.6.10

OS & Version: CentOS 7.9

Chain/Network: ETH

When calling the trace_block API to request data for block 21030627, an error was returned:
```
{'jsonrpc': '2.0', 'id': 1, 'error': {'code': -32000, 'message': 'first run for txIndex 85 error: insufficient funds for gas * price + value: address 0x0b92619DdE55C0cbf828d32993a7fB004E00c84B have 1414044108792614043 want 1414044806692614043'}}
```

After deleting all node data and re-synchronizing, the error still returns with the same block 21030627.

**Comment by VBulikov** [comment]
relates to #12525

**Comment by yperbasis** [comment]
Fixed by PR #13125/#13128/#13129
